docs/api/instruments/anapico/index.srt

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.anapico
```

Comment:
This file extension should be rst, not srt. :-)

Reply:
Sorry...

Reply:
no problem. btw, I think you forgot to delete the old .srt files. ;-)

Reply:
Done! Pardon.